strawberry/federation/field.py

```diff
@@ -52,6 +50,10 @@ def field(
     provides: Optional[List[str]] = None,
     requires: Optional[List[str]] = None,
     external: bool = False,
+    shareable: bool = False,
+    tag: Optional[List[str]] = None,
+    override: Optional[List[str]] = None,
```

Comment:
```suggestion
    tags: Optional[Iterable[str]] = (),
    overrides: Optional[Iterable[str]] = (),
```

Reply:
resolved, with the exception of "overrides" as it is singular rather than plural -- not repeatable (or something)